### PR TITLE
added new login instructions

### PR DESCRIPTION
**Pull request by urikirsh**
Added an instruction page for how to set up an account in cognito

### CODE REVIEW

File: docs/prover/new_login.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+============
+
+We are happy to announce that we are migrating to a new user management system that will use your email to access the Certora Prover dashboard. 
+Decoupling your account from a specific key will allow us to introduce features which will dramatically improve your developer experience in the future.  
+
+The migration to the new system is set to occur on Sunday, November 5th. You will receive a welcome email with an initial password and a link to the new login page.
+
+Please follow these instructions:
+- Once you receive the welcome email, log out of your accounts dashboard at Certora website
+- Click the `LOGIN TO CERTORA` link from the email. You will be redirected to the new login page and will be asked to change the initial password to activate your account.
```
Comment (shellygr) [inline docs/prover/new_login.md:11]:
```suggestion
- Click the `LOGIN TO CERTORA` link from the email. You will be redirected to the new login page and will be asked to change the initial password to activate your account
```

File: docs/prover/new_login.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+- Follow the password prompts to authenticate the new password
+- After the initial login, replace the current Certora’s login page in your bookmarks with: https://prover.certora.com
+
+
+Once the steps above are done, your email address is going to be used as your username to login to the Certora’s website (Prover dashboard) and it's going to replace the Certora Prover key during login.
```
Comment (shellygr) [inline docs/prover/new_login.md:16]:
```suggestion
Once the steps above are done, your email address is going to be used as your username to login to Certora’s website (Prover dashboard) and it's going to replace the Certora Prover key during login.
```

File: spelling_wordlist.txt [hunk 3]
```diff
@@ -34,6 +34,7 @@ bugfix
 bytecode
 bytecodes
 callee
+certora
```
Comment (shellygr) [inline spelling_wordlist.txt:37]: weird this is necessary...

**Review by shellygr — APPROVED** [review]
few nits

File: spelling_wordlist.txt (hunk 3 above)
Comment (urikirsh) [reply spelling_wordlist.txt:37]: it complains about `https://www.certora.com/...`

**Pull request merged**